### PR TITLE
Add risk functionality

### PR DESCRIPTION
**Pull request by uhakdt**
- Added "Risk" column to Review, Assess and Verify tables
- Added "Risk" property to the EF Models
- When progressing the task we assigned "risk"

Co-authored-by: Samir Hassoun <51328831+SamirHassoun@users.noreply.github.com>
Co-authored-by: Umar Khan <umar.khan@ukho,.gov.uk>

### CODE REVIEW

**Comment by UKHOGreg** [comment]
Pipelines are failing

**Review by UKHOGreg — APPROVED** [review]
🚀

File: src/Databases/WorkflowDatabase/Tables/DbAssessmentAssessData.sql [hunk 1]
```diff
@@ -4,7 +4,8 @@
 	[ProcessId] INT NOT NULL, 
 	[Ion] NVARCHAR(50) NULL, 
 	[ActivityCode] NVARCHAR(50) NULL, 
-    [Complexity] NVARCHAR(50) NULL, 
+    [Complexity] NVARCHAR(50) NULL,
```
Comment (kevinpearce) [inline src/Databases/WorkflowDatabase/Tables/DbAssessmentAssessData.sql:7]: Out of format indentation?

Comment (kevinpearce) [reply src/Databases/WorkflowDatabase/Tables/DbAssessmentAssessData.sql:7]: Discussed with SH - codebase is fine, Github appearance only.

**Review by kevinpearce — APPROVED** [review]

**Pull request merged**